README.md

```diff
@@ -109,6 +115,10 @@ Example: `"BestMod.QMod.Patch"`
   "Author": "Awesome Guy",
   "Version": "1.0.0",
   "Dependencies": [ "DependencyModID" ],
+  "VersionDependencies": { 
+    "VeryImportantMod": "1.2.x", 
+    "AnotherVeryImportantMod": "2.1.2 - 2.3.2" 
```

Comment:
The samples provided here should showcase the most common usage of the feature.  
In most cases, dependencies are "_this specific version or later_".  
Make the example something like that.